client/src/scss/_variables.scss

```diff
@@ -27,3 +27,7 @@ $z-index-levels: (
 @function static-url($path) {
   @return url($static-url + $path);
 }
+
+// Responsivity
+
+$mobile-break-point: 500px;
```

Comment:
Add comment should be same as in js, specify place

Reply:
clarify?

Reply:
// Breakpoint in pixels same as in js check in file: x.js